auth-web/src/views/auth/staff/ReviewAccountView.vue

```diff
@@ -110,409 +117,538 @@ import StaffModuleStore from '@/store/modules/staff'
-  @StaffModule.State('accountUnderReviewAdminContact') public accountUnderReviewAdminContact!: Contact
-  @StaffModule.State('accountUnderReviewAffidavitInfo') public accountUnderReviewAffidavitInfo!: AffidavitInformation
-
-  @StaffModule.Getter('accountNotaryName') public accountNotaryName!: string
-
-  @StaffModule.Action('syncTaskUnderReview') public syncTaskUnderReview!: (task:Task) => Promise<void>
-  @StaffModule.Action('approveAccountUnderReview') public approveAccountUnderReview!: (task:Task) => Promise<void>
-  @StaffModule.Action('rejectorOnHoldAccountUnderReview') public rejectorOnHoldAccountUnderReview!: (task:any) => Promise<void>
-
-  @orgModule.Action('fetchCurrentOrganizationGLInfo') public fetchCurrentOrganizationGLInfo!:(accountId: number) =>Promise<any>
-  @orgModule.State('currentOrgGLInfo') public currentOrgGLInfo!: GLInfo
-  @orgModule.Action('fetchOrgProductFeeCodes') public fetchOrgProductFeeCodes!:() =>Promise<OrgProductFeeCode>
-  @orgModule.Action('getOrgProducts') public getOrgProducts!:(accountId: number) =>Promise<OrgProduct[]>
-  @orgModule.Action('createAccountFees') public createAccountFees!:(accoundId:number) =>Promise<any>
-  @orgModule.Action('syncCurrentAccountFees') public syncCurrentAccountFees!:(accoundId:number) =>Promise<AccountFee[]>
-  @orgModule.Mutation('resetCurrentAccountFees') public resetCurrentAccountFees!:() =>void
-  @orgModule.Action('updateOrganizationAccessType') updateOrganizationAccessType!:({ accessType, orgId, syncOrg }) => Promise<boolean>
-
-  @CodesModule.Action('getOnholdReasonCodes') public getOnholdReasonCodes!: () => Promise<Code[]>
-  @CodesModule.State('onholdReasonCodes') private readonly onholdReasonCodes!: Code[]
-
-  private staffStore = getModule(StaffModuleStore, this.$store)
-  public isLoading = true
-  public isSaving = false
-
-  private readonly pagesEnum = Pages
-  private readonly TaskRelationshipStatusEnum = TaskRelationshipStatus
-
-  private isConfirmationModal:boolean = false
-  private isRejectModal:boolean = false
-  private isOnHoldModal:boolean = false
-  public task :Task
-  public taskRelationshipType:string = ''
-  private productFeeFormValid: boolean = false
-  private viewOnly = DisplayModeValues.VIEW_ONLY
-  accountInfoAccessType: AccessType = null
-  accountInfoValid = true
-  showAccountInfoValidations = false
-
-  $refs: {
-    accessRequest: AccessRequestModal,
-    productFeeRef: HTMLFormElement
-  }
+    AccessRequestModal,
+    AccountStatusTab
+  },
+  setup (props) {
+    const store = useStore()
+    const instance = getCurrentInstance()
+    const staffStore = getModule(StaffModuleStore, store)
+    const isLoading: Ref<boolean> = ref(true)
+    const isSaving: Ref<boolean> = ref(false)
+    const pagesEnum = Pages
+    const TaskRelationshipStatusEnum = TaskRelationshipStatus
+    const isConfirmationModal: Ref<boolean> = ref(false)
+    const isRejectModal: Ref<boolean> = ref(false)
```

Comment:
Could use `reactive` instead of `ref`, makes it a bit easier with not having to use `.value`